posts/atomic-design-bootstrap.md

```diff
@@ -0,0 +1,154 @@
+
+However, it is difficult to use Bootstrap in an Atomic Design approach. We quickly lose the consistency of our application in favor of Bootstrap's graphical identity.
+
+I want to share with you my experience of _Bootstrap_ so that you know how to make it cohabit with your own styles rather than fighting against it.
+
+# But, what is Bootstrap ?
+
+> What is Bootstrap ?
+
+This question may seem trivial at first sight, but it is not!
+
+From the official website, it is a framework for building [responsive](https://alistapart.com/article/responsive-web-design), [mobile-first](https://www.lukew.com/ff/entry.asp?933) web applications. According to them, it's the world's most popular front-end component library on the web and it's based on:
+
+* HTML structure
+* CSS style file (also Sass sources)
+* JavaScript for component behaviour (display a modal, alert messages, …) with libraries like jQuery.
+
+To help us, _Bootstrap_ give us a grid system to organise components using columns that will be adapted to the screen size (we will talk about it later in the blog post) and a lot of components ready to use.
+
+In summary, on one hand, _Bootstrap_ proposes a Pattern Library. This is an organised representation of reusable components using code snippets, design guidelines and use cases. On the other hand, it is a set of mechanic components (Alerts, Toggle, Carousel, Collapse, Modal …) made with [JavaScript, jQuery and Popper](https://getbootstrap.com/docs/4.4/getting-started/introduction/#js)
+
+![Descriptive schema about a Pattern Library](https://raw.githubusercontent.com/gnuk/blog-usa/atomic-bootstrap/images/2020/01/atomic-design-bootstrap-pattern-library.png)
+_Pattern Library_
+
+The rest of this blog post will focus on the `Pattern Library` aspect.
+
+# Use the strength of Bootstrap
+
+As reminder, _Atomic Design_ allows you to create and assemble graphical components by granularity.
+
+The _atoms_ combine to form _molecules_, which themselves assemble into _organisms_ to form templates which constitute the models of our pages.
+
+The global idea is to define a graphic design. _Bootstrap_ being an existing Pattern Library, there may be a risk of conflict between the _Bootstrap_ choices and ours.
+
+Nevertheless, using this type of framework brings advantages (reusable components, graphic coherence, numerous contributors ...); it may be difficult to remove it.
+
+That's why I'm going to show you several examples to show you how to make _Atomic Design_ and _Bootstrap_ coexist.
+
+# Try to not use Bootstrap components
+
+First of all, I recommend you to avoid the use of _Bootstrap_ [Components](https://getbootstrap.com/docs/4.4/components). Why? Because they are graphic elements with a strong impact on what the application is doing.
```

Comment:
```suggestion
First of all, I recommend that you avoid the use of _Bootstrap_ [Components](https://getbootstrap.com/docs/4.4/components). Why? Because they are graphic elements with a strong impact on what the application is doing.
```